modules/currency.ts

```diff
@@ -164,6 +165,15 @@ function loadRates() {
   }
 }
 
+declare module '../src/prebidGlobal' {
+    interface PrebidJS {
```

Comment:
There's already a `PrebidJS` interface in the `prebidGlobal.ts` . Why not put this method there?